activestorage/test/models/blob_test.rb

```diff
@@ -177,7 +177,7 @@ class ActiveStorage::BlobTest < ActiveSupport::TestCase
   test "purge fails when attachments exist" do
     create_blob.tap do |blob|
       User.create! name: "DHH", avatar: blob
-      assert_raises(ActiveRecord::InvalidForeignKey) { blob.purge }
+      assert_nothing_raised { blob.purge }
```

Comment:
Let’s assert that the blob isn’t deleted instead:

```ruby
assert_no_difference(-> { ActiveStorage::Blob.count }) { blob.purge }
```

Finally, please update the name of the test to reflect that `purge` no longer fails when attachments exist: `"purge does nothing when attachments exist"`.

Reply:
Oh okay, that seems to be more appropriate! 👍